src/public/preload.js

```diff
@@ -15,6 +15,29 @@ events.forEach(function (e) {
     });
 });
 
+const userPresenceControl = () => {
+    UserPresence.stopTimer(); //stop userpresence control
+    const AWAY_TIME = 300000; // 5 min
+    const INTERVAL = 10000; // 10s
+    let userOnline = true;
+    setInterval(() => {
+        try {
+            const idleTime = ipcRenderer.sendSync('getSystemIdleTime');
+            if (idleTime < AWAY_TIME && !userOnline) {
+                Meteor.call('UserPresence:online');
+                userOnline = true;
+            }
+            if (idleTime >= AWAY_TIME && userOnline) {
+                if (!userOnline) { return; }
```

Comment:
this condition will never happen, since you're testing `userOnline` on the line before